README.md

```diff
@@ -42,7 +42,7 @@
 I have created/will be creating the following courses/tutorials/learning series on my YouTube channel -
 
 1. Being-Better
-2. AAM (🥭) 
+2. AAM (🥭)
```

Comment:
```suggestion
2. AAM (🥭) 
```

No need for this, doesn't really matter
Rest LGTM,
Thanks for the contribution